src/main/java/com/workoutbuilder/enterprise/controller/WorkoutBuilderController.java

```diff
@@ -272,7 +276,7 @@ public StoredExercise createExercise(@RequestBody StoredExercise exercise) {
         try {
             newExercise = storedExerciseService.saveStoredExercise(exercise);
         } catch (Exception e) {
-            //TODO LOG ERROR
+            LOGGER.error("Unable to create exercise with id: " + exercise.getId() + " | error: " + e.getMessage(), e);
```

Comment:
I added a log message. I saw a TODO comment to log the error, so I thought I'd try implementing the logger. It's purpose is to provide some information incase an error occurs. 

It explains the importance of logging in this article: https://www.baeldung.com/java-clean-code